src/modules/PublicLab.MapModule.js

```diff
@@ -23,13 +23,17 @@ module.exports = PublicLab.MapModule = PublicLab.Module.extend({
          lngId: 'lng'
        }
      }
+     var token = "pk.eyJ1Ijoianl3YXJyZW4iLCJhIjoiVzVZcGg3NCJ9.BJ6ArUPuTs1JT9Ssu3K8ig";
+
+     options.tileLayerUrl = 'https://api.mapbox.com/styles/v1/mapbox/outdoors-v11/tiles/{z}/{x}/{y}?access_token=' + token;
```

Comment:
Hey i like this tileLayer :)
But i think we should also accept this as an argument when we initialize the editor object to overwrite this. What do you think? Thanks!

Reply:
Sounds good...Adding that. Thanks